app/serializers/api/admin/enterprise_serializer.rb

```diff
@@ -13,7 +13,7 @@ class EnterpriseSerializer < ActiveModel::Serializer
                  :default_tag_group, :require_login, :allow_guest_orders, :allow_order_changes,
                  :logo, :promo_image, :terms_and_conditions,
                  :terms_and_conditions_file_name, :terms_and_conditions_updated_at,
-                 :preferred_invoice_order_by_supplier
+                 :preferred_invoice_order_by_supplier, :visible
```

Comment:
I don't see this used anywhere? The serialized values are only needed in Javascript when querying the API, I think.

Reply:
When the three checkboxes are rendered on the admin page, it needs to check to see what the value of `Enterprise.visible` and compares it for each of the checkboxes [here](https://github.com/openfoodfoundation/openfoodnetwork/pull/9087/files#diff-1722d363f014b52f232ff3cf935c0c52b53a6f36ec10fe14f6c921b3b45cec25R45). Whichever matches to the correct value, is then given the checked html attribute. So the visible attribute needs to be exposed now